tests/cases/fourslash/server/pasteEdits_namespaceImport.ts

```diff
@@ -0,0 +1,46 @@
+/// <reference path="../fourslash.ts" />
+
+// @Filename: /home/src/workspaces/project/folder/c.ts
+//// [||]
+
+
+// @Filename: /home/src/workspaces/project/a.ts
+//// const abc = 10;
+//// const def = 20;
+//// export interface testInterface {
+////     abc: number;
+////     def: number;
+//// }
+
+// @Filename: /home/src/workspaces/project/b.ts
+//// import * as test from "./a";
+////
+//// [|function foo(abc: test.testInterface, def: test.testInterface) {

```

Comment:
```suggestion
//// [|function foo(abc: test.abc, def: test.def) {
```
to match the `pastesText` arguments